src/snowflake/connector/network.py

```diff
@@ -246,6 +248,46 @@ def __call__(self, r):
         return r
 
 
+class SessionPool(object):
```

Comment:
I'm pretty sure this is implicitly done for all classes since Python 3
```suggestion
class SessionPool:
```